### PR TITLE
improve current ras case and add lmce test cases

### PR DESCRIPTION
**Pull request by laifryiee**

### CODE REVIEW

File: BM/ras/ras_common.sh [hunk 1]
```diff
@@ -21,3 +25,47 @@ pkg_check_install() {
...
+    done
+    if [ "$daemon" -eq 0 ] || [ "$logfile" -eq 0 ]; then
+        test_print_trc "mcelog service is not properly configured, reload the service."
+        kill -9 $(pgrep -x mcelog)
+        /usr/sbin/mcelog --ignorenodev --daemon --logfile=${MCELOG_LOGFILE}
+    fi
+}
+
+# disable MCE CMCI
+disable_cmci() {
```
Comment (ysun) [inline BM/ras/ras_common.sh:55]: Hi @laifryiee  Please use two spaces for indentation for shell code. And please refine the whole file. Thanks

File: BM/ras/ras_common.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# Copyright (c) 2024 Intel Corporation
+# Author: Yi Lai <yi1.lai@intel.com>
+# @Desc  Common functions used in ras test suite
+
+cd "$(dirname "$0")" 2>/dev/null || exit 1
+source ../.env
+
+# check whether one package is installed, if not, install the package
+# Usage: pkg_check_install pkg_name_1 pkg_name_2 pkg_name_x
+pkg_check_install() {
```
Comment (ysun) [inline BM/ras/ras_common.sh:16]: Also please fix this indentation with two spaces.

**Review by ysun — CHANGES_REQUESTED** [review]
Please fix the two commits. Others look good to me. thanks

File: BM/ras/ras_common.sh (hunk 1 above)
Comment (laifryiee) [reply BM/ras/ras_common.sh:55]: Hi, @ysun  all indentation issues are resolved. Please help take a look again.

**Pull request merged**